types/resource.go

```diff
@@ -198,10 +198,23 @@ const (
 	NodeStateDown = NodeState("down")
 )
 
+type NodeConditionType string
+
+const (
+	NodeConditionTypeMountPropagation = "MountPropagation"
+	NodeConditionTypeNodeState        = "NodeState"
+	NodeConditionTypeDiskState        = "DiskState"
```

Comment:
Disk conditions should be in the disk status for each disk. And don't use `state` kind of description. Condition can be `True`, `False`, `Unknown`. So for the disk it can be `Schedulable`.